suite-common/wallet-config/src/networksConfig.ts

```diff
@@ -434,7 +434,12 @@ export const networks = {
         },
         customBackends: ['solana'],
         accountTypes: {
-            // TODO(vl): accounts and paths
+            bip44Change: {
+                bip43Path: "m/44'/501'/i'/0'",
+            },
+            bip44Root: {
+                bip43Path: "m/44'/501'",
+            },
```

Comment:
So basically two new paths, this [article](https://help.phantom.app/hc/en-us/articles/12988493966227-What-derivation-paths-does-Phantom-wallet-support-) might give you some context.